src/v2/providers/storage.ts

```diff
@@ -29,6 +29,7 @@ import * as options from '../options';
 /**
  * An object within Google Cloud Storage.
  * Ref: https://github.com/googleapis/google-cloudevents-nodejs/blob/main/cloud/storage/v1/StorageObjectData.ts
+ * @beta
```

Comment:
we should yank beta per our offline discussion

Reply:
Done!